File: python/1061.py
import math

line1=input().split(" ")
line2=input().split(" ")
line3=input().split(" ")
line4=input().split(" ")
dia1=int(line1[1])
dia2=int(line3[1])
hora1=int(line2[0])
hora2=int(line4[0])
minuto1=int(line2[2])
minuto2=int(line4[2])
segundo1=int(line2[4])
segundo2=int(line4[4])

def convertTime(d,h,m,s):
    return (d*86400)+(h*3600)+(m*60)+s

initial=convertTime(dia1,hora1,minuto1,segundo1)
final=convertTime(dia2,hora2,minuto2,segundo2)
toSec=final-initial

dias=toSec/86400
toSec%=86400
horas=toSec/3600
toSec%=3600
minutos=toSec/60
toSec%=60
segundos=toSec

print(str(math.floor(dias))+ ' dia(s)')
print(str(math.floor(horas))+ ' hora(s)')
print(str(math.floor(minutos))+ ' minuto(s)')
print(str(segundos)+ ' segundo(s)')

File: javascript/1061.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var [diaStr1, tempoStr1, diaStr2, tempoStr2] = lines;
var splitDia1 = diaStr1.split(" ")
var splitTempo1 = tempoStr1.split(" ")
var splitDia2 = diaStr2.split(" ")
var splitTempo2 = tempoStr2.split(" ")

var dia1 = parseInt(splitDia1[1])
var dia2 = parseInt(splitDia2[1])
var hora1 = parseInt(splitTempo1[0])
var hora2 = parseInt(splitTempo2[0])
var minutos1 = parseInt(splitTempo1[2])
var minutos2 = parseInt(splitTempo2[2])
var segundos1 = parseInt(splitTempo1[4])
var segundos2 = parseInt(splitTempo2[4])

function convertTime(d,h,m,s){
    return (d*86400)+(h*3600)+(m*60)+s;
}

var initialSecs = convertTime(dia1,hora1,minutos1,segundos1)
var finalSecs = convertTime(dia2,hora2,minutos2,segundos2)
var defSecs = finalSecs - initialSecs

var dias = defSecs/86400
defSecs%=86400
var horas = defSecs/3600
defSecs%=3600
var minutos = defSecs/60
defSecs%=60
console.log(`${parseInt(dias)} dia(s)`)
console.log(`${parseInt(horas)} hora(s)`)
console.log(`${parseInt(minutos)} minuto(s)`)
console.log(`${parseInt(defSecs)} segundo(s)`)